os/arch/arm/src/amebasmart/Make.defs

```diff
@@ -233,10 +233,11 @@ CHIP_CSRCS += amebasmart_timer_lowerhalf.c
 CHIP_CSRCS += amebasmart_rtc_lowerhalf.c
 CHIP_CSRCS += amebasmart_rtc.c
 CHIP_CSRCS += amebasmart_watchdog_lowerhalf.c
-#CHIP_CSRCS += amebasmart_i2c.c
-#CHIP_CSRCS += amebasmart_i2s.c
-#CHIP_CSRCS += amebasmart_spi.c
-
+CHIP_CSRCS += amebasmart_i2c.c
+CHIP_CSRCS += amebasmart_spi.c
```

Comment:
How about adding conditionals for them like i2s?
```
ifeq ($(CONFIG_AMEBASMART_I2C),y)
CHIP.....+=...i2c.c
endif
ifeq(....SPI),y)
CHIP...+=...spi.c
endif
```

Reply:
This is the my previous comment. I suggest the same conditionals in the boot.c for init function calls.